main.c

```diff
@@ -505,7 +505,7 @@ int __attribute__((used)) main(void) {
         }
         if (exit_code == PYEXEC_FORCED_EXIT) {
             if (!first_run) {
-                serial_write_compressed(translate("soft reboot\n"));
+                serial_write_compressed(translate("\n\n------ soft reboot ------\n"));
```

Comment:
Why are you changing this? I don't think you should because some tools like the pyboard loader look for this string.

Reply:
With the current implementation it is not clear when the `soft reboot` happened as it doesn't get printed out in a newline. 

Following code can work without breaking compatibility:
```suggestion
                serial_write("\n\n");
                serial_write_compressed(translate("soft reboot\n"));
```
I can just remove it though, if you say so.